fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/controller/FinTechAccountInformationImpl.java

```diff
@@ -46,12 +46,15 @@
                                                                             LocalDate dateFrom, LocalDate dateTo,
                                                                             String entryReferenceFrom, String bookingStatus, Boolean deltaList) {
         if (!authorizeService.isAuthorized(xsrfToken, null)) {
+            log.warn("Request was failed: Xsrf Token is wrong or user are not authorized!");
             return new ResponseEntity<>(HttpStatus.UNAUTHORIZED);
         }
 
         ContextInformation contextInformation = new ContextInformation(xRequestID);
         SessionEntity sessionEntity = authorizeService.getByXsrfToken(xsrfToken);
 
+        redirectHandlerService.registerRedirectUrlForSession(xsrfToken, fintechRedirectURLOK, fintechRedirectURLNOK);
```

Comment:
Probably, Open banking should be called with different callback URLs. 
fintechRedirectURLOK, fintechRedirectURLNOK are URLs to notify whether consent was created or not.
They MUST include `redirectCode`, so that you will be able to load corresponding session from the database.

In the current implementation, fintechRedirectURLOK and fintechRedirectURLOK passed to OPBA are the same.

I think fintechRedirectURLOK that is passed to OPBA can be something like:
`fintechRedirectURLOK + &redirectCode` if frontend is going to pass URLs to this API